pkgs/by-name/wa/warp-terminal/versions.json

```diff
@@ -1,10 +1,10 @@
 {
   "darwin": {
-    "hash": "sha256-VHyEE0SziwDAzlv8VLt08tMXb20sqxTSj64hC+FyjUw=",
-    "version": "0.2024.03.05.08.02.stable_01"
+    "hash": "sha256-GgaRlROSWYZgFlH0bH6PTnRE3L/bb0kX0P6m7nmPlsY=",
+    "version": "0.2024.03.12.08.02.stable_01"
   },
   "linux": {
-    "hash": "sha256-CI1bzdFles9XNvqmkyNq9zJBf4P6HF8QIo1FsSDydjQ=",
-    "version": "0.2024.03.05.08.02.stable_01"
+    "hash": "sha256-id+EWr+U8i6Y6ELu7GmDOfh2nucyR+8qBoxcMtKIkj8=",
```

Comment:
```suggestion
    "hash": "sha256-9reFBIu32TzxE46c3PBVzkZYaMV4HVDASvTAVQltYN0=",
```

`nix-prefetch-url` in the update script is uncompressing the archive which leads to generating the wrong hash.

PR to fix update script to workaround issue is: https://github.com/NixOS/nixpkgs/pull/296333

```console
$ nix-build --expr 'with import ./.{}; fetchurl { url = "https://releases.warp.dev/stable/v0.2024.03.12.08.02.stable_01/warp-terminal-v0.2024.03.12.08.02.stable_01-1-x86_64.pkg.tar.zst"; }'
his derivation will be built:
  /nix/store/rwy0v85qq0gsvlg6762jsly6lk61m879-warp-terminal-v0.2024.03.12.08.02.stable_01-1-x86_64.pkg.tar.zst.drv
building '/nix/store/rwy0v85qq0gsvlg6762jsly6lk61m879-warp-terminal-v0.2024.03.12.08.02.stable_01-1-x86_64.pkg.tar.zst.drv'...

trying https://releases.warp.dev/stable/v0.2024.03.12.08.02.stable_01/warp-terminal-v0.2024.03.12.08.02.stable_01-1-x86_64.pkg.tar.zst
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 33.7M    0 33.7M    0     0  31.5M      0 --:--:--  0:00:01 --:--:-- 31.6M
error: hash mismatch in fixed-output derivation '/nix/store/rwy0v85qq0gsvlg6762jsly6lk61m879-warp-terminal-v0.2024.03.12.08.02.stable_01-1-x86_64.pkg.tar.zst.drv':
         specified: sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=
            got:    sha256-9reFBIu32TzxE46c3PBVzkZYaMV4HVDASvTAVQltYN0=
```

Reply:
fixed